tchannel/peer_heap.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+from .container import heap
+from .container.heap import HeapOperation
+
+
+class PeerHeap(HeapOperation):
+    def __init__(self):
+        self.peers = []
```

Comment:
Since you'll be adding and removing items from this all the time, consider using [deque](https://docs.python.org/2/library/collections.html#collections.deque)